java/clients/src/test/resources/reports/compaction/job/standard/detailed/mixedJobs.txt

```diff
@@ -12,10 +12,9 @@ Start Update Time: 2022-09-20T13:34:12.123Z
 Finish Time: 2022-09-20T13:35:12.001Z
 Finish Update Time: 2022-09-20T13:35:12.123Z
 Duration: 1m
-Records Read: 0
-Records Written: 0
-Read Rate (reads per second): 0.00
-Write Rate (writes per second): 0.00
+Run failed, reasons:
+Something went wrong
+More details
```

Comment:
Perhaps we could prepend a dash to the reasons to distinguish them from the rest of the detailed text?